src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2140,6 +2140,17 @@ void processContextElement
 
   while (cursor->more())
   {
+    if (strcasecmp(action.c_str(), "appendonly") == 0)
```

Comment:
Should the attribute be checked also? I mean, from my point of view, if the entity is E and have attributes A, B, C an updateContext on E-D is a legal "appendonly", given that E doesn't have the D attribute. However, this if clause precludes that usage...